lib/paper_trail/frameworks/rails/railtie.rb

```diff
@@ -25,6 +25,10 @@ class Railtie < ::Rails::Railtie
       ActiveSupport.on_load(:active_record) do
         require "paper_trail/frameworks/active_record"
       end
+
+      if ::Rails::VERSION::STRING >= "7.1"
```

Comment:
```suggestion
      if Gem::Version.new(::Rails::VERSION::STRING) >= Gem::Version.new("7.1")
```

Without this small change, it will break when Rails reaches version 10.0 :)